src/current/cockroachcloud/production-checklist.md

```diff
@@ -24,7 +24,9 @@ After an application establishes a connection to {{ site.data.products.db }}, th
 
 Make sure connection validation and retry logic is used by your application. Validating and retrying connections is typically handled by the driver, framework, or the connection pool used by an application. For guidance on connection pool sizing, connection validation, and connection retry logic, see [Use Connection Pools](../{{site.current_cloud_version}}/connection-pooling.html).
 
-{% include common/transaction-retries.md %}
+## Transaction retries
```

Comment:
@taroface Wanted to give you a heads-up. I saw you made the equivalent change in the [v23.1 docs](https://github.com/cockroachdb/docs/pull/16823), so I reflected the same change in the CC docs, in case you need to make changes to this again in the future.